cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,48 @@
 /// <reference types='cypress' />
 
-describe('', () => {
-  before(() => {
+import HomeAndCataloguePageObject
+  from '../support/pages/homeСatalogue.pageObject';
+import CartFormPageObject from '../support/pages/cartForm.pageObject';
+import faker from 'faker';
 
-  });
+const homePage = new HomeAndCataloguePageObject();
+const cartForm = new CartFormPageObject();
+const fakeData = {
+  name: faker.name.findName(),
+  country: faker.address.country(),
+  city: faker.address.city(),
+  creditCard: faker.finance.creditCardNumber(),
+  month: faker.date.month(),
+  year: faker.date.future().getFullYear()
+};
 
-  it('', () => {
+describe('Purchase flow', () => {
+  before(() => {
+    homePage.visit();
+  });
 
+  it('should order a laptop', () => {
+    homePage.clickOnCategory('Laptops');
+    homePage.clickOnProduct('Sony vaio i7');
+    homePage.clickOnBtn('Add to cart');
```

Comment:
After clicking the product name we are redirected to the product page `/prod.html`, so this method should come from a different pageObject file, e.g. `productPageObject.js`.